### ISSUE TITLE
Pulmonary parenchymal band

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6703]
**Preferred term label:**
Pulmonary parenchymal band

**Synonyms**

**Definition (free text, please give PubMed ID)**
A parenchymal band is a linear opacity, usually 1-3 mm thick and up to 5 cm long that usually extends to the visceral pleura
(which is often thickened and may be retracted at the site of contact). It reflects pleuroparenchymal fibrosis and is usually associated with distortion of the lung architecture. Parenchymal bands are most frequently encountered in individuals who have been exposed to asbestos.

PMID:18195376

Comment
See Figure 48 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
